src/VirtualAccount.php

```diff
@@ -19,15 +19,15 @@ public function all($options = array())
         return parent::all($options);
     }
 
-    public function close()
+    public function close($id)
```

Comment:
@neera11 I think the call to this method will look like $this->virtualAccount->close(). pls verify from readme once. Also changing the structure will lead to integration changes, so i think method signature should be kept same.

Reply:
corrected method signature